pkgs/top-level/all-packages.nix

```diff
@@ -1913,6 +1913,9 @@ with pkgs;
 
   gopass-summon-provider = callPackage ../tools/security/gopass/summon.nix { };
 
+  kanidm_1_3 = callPackage ../by-name/ka/kanidm/1_3.nix { };
+  kanidm_1_4 = callPackage ../by-name/ka/kanidm/1_4.nix { };
+
```

Comment:
Can we move this to around line ~11700 (category 'servers')? We already have `kanidmWithSecretProvisioning` declared there

Reply:
Done, I also added provisioning packages for the new versioning.